src/ec/packages/ec-component-description-list/description-list.story.js

```diff
@@ -22,20 +26,35 @@ const prepareList = data => {
   }
   data.items.forEach((item, i) => {
     if (Array.isArray(item.term)) {
```

Comment:
Array and single item are checked. What about an object?

Reply:
But that is not a use case in this component:
`- "term" (string or array of string)`